tests/common/integration_features/test_vulnerabilities_integration.py

```diff
@@ -0,0 +1,205 @@
+import os
+import unittest
+
+from checkov.common.bridgecrew.check_type import CheckType
+from checkov.common.bridgecrew.integration_features.features.vulnerabilities_integration import \
+    VulnerabilitiesIntegration, NORMALIZE_PREFIX
+from checkov.common.bridgecrew.platform_integration import BcPlatformIntegration
+from checkov.common.output.record import Record, SCA_PACKAGE_SCAN_CHECK_NAME
+from checkov.common.output.report import Report
+from checkov.sast.consts import SastLanguages
+from checkov.sast.report import SastReport
+
+
+class TestVulnerabilitiesIntegration(unittest.TestCase):
```

Comment:
can we use a `pytest` suite instead of `unittest`? you can then mock env variables with decorators and make tests more readable :)

Reply:
I want to be consistent with all other tests in this directory.